CalculateTotal.js

```diff
@@ -0,0 +1,42 @@
+function getOrderTotal(items) {
+  var total = 0;
+  items.map((x) => {
+    total += x.cost * x.quantity;
+  });
+
+  // console.log("total", total);
+  return total;
+}
+
+var calculateSalesTax = function(total) {
+  return total + (total * 15) / 100;
+};
+
+function calculateTotal(itemsInCart) {
```

Comment:
There was inconsistent use of `var` and `let` here. As per the comment above, we can use `const` here.